vista_2d/vista_2d_tutorial_monai.ipynb

```diff
@@ -0,0 +1,1405 @@
+{
```

Comment:
<p>Could you please help remove the output for this cell? But include the graph in the next cell outout, thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Sure, let me fix it with the next run.</p>

Reply:
Sure.